### PR TITLE
[full-ci] Replace web-app-admin-settings action mixins with composables

### PR DESCRIPTION
**Pull request by dschmidt**
# DEPENDS ON https://github.com/owncloud/web/pull/8659

## Description
Port action mixins in web-app-admin-settings to composables.

## Related Issue

- Fixes <issue_link>

## Motivation and Context
We want to get rid of all composables.

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/33955/11/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33955/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-alice-2023-3-21-05-13-41.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33955/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-brian-2023-3-21-05-14-12.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33955/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-carol-2023-3-21-05-14-19.zip 


</p></details>

File: packages/web-app-admin-settings/src/composables/actions/general/useGeneralActionsResetLogo.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  const handler = async () => {
+    try {
+      const httpClient = clientService.httpAuthenticated
+      await httpClient.delete('/branding/logo')
+      store.dispatch('showMessage', {
+        title: $gettext('Logo was reset successfully. Reloading page...')
+      })
+      setTimeout(() => {
+        router.go(0)
+        router
```
Comment (JammingBen) [inline packages/web-app-admin-settings/src/composables/actions/general/useGeneralActionsResetLogo.ts:23]:
```suggestion
```

File: packages/web-app-admin-settings/src/composables/actions/users/useUserActionsEdit.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { eventBus } from 'web-pkg'
+import { SideBarEventTopics } from 'web-pkg/src/composables/sideBar'
+import { useGettext } from 'vue3-gettext'
+import { computed } from 'vue'
+import { Action } from 'web-pkg/src/composables/actions'
+
+export const useUserActionsEdit = () => {
+  const { $gettext } = useGettext()
+
+  const actions = computed((): Action[] => [
```
Comment (JammingBen) [inline packages/web-app-admin-settings/src/composables/actions/users/useUserActionsEdit.ts:10]: Why not `UserAction`? Then you don't need to cast `resources as unknown[]` down below.

```suggestion
  const actions = computed((): UserAction[] => [
```

**Review by JammingBen — COMMENTED** [review]
Minor nitpicks, rest LGTM 👍

File: packages/web-app-admin-settings/tests/unit/composables/actions/useGeneralActionsResetLogo.spec.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  RouteLocation,
+  mockAxiosResolve,
+  mockAxiosReject,
+  getComposableWrapper
+} from 'web-test-helpers'
+
+jest.useFakeTimers()
+
+describe('resetLogo', () => {
+  describe('method "$_resetLogo_reset"', () => {
```
Comment (kulmann) [inline packages/web-app-admin-settings/tests/unit/composables/actions/useGeneralActionsResetLogo.spec.ts:17]: `$_resetLogo_reset` :trollface:

**Review by kulmann — APPROVED** [review]
💪

**Pull request merged**